client/boot/index.js

```diff
@@ -174,7 +174,7 @@ function renderLayout( reduxStore ) {
 	let props = { focus: layoutFocus };
 
 	if ( user.get() ) {
-		Object.assign( props, {}, { user, sites, nuxWelcome, translatorInvitation, pushNotifications } );
+		Object.assign( props, {}, { user, sites, nuxWelcome, translatorInvitation } );
```

Comment:
Why the first `{}` in the list of sources for `Object.assign`? (I understand this particular bit existed prior to this diff.)